portfolio/src/main/webapp/leave-comment/leave-comment.html

```diff
@@ -12,19 +14,25 @@
     <link rel="stylesheet" href="leave-comment.css">
   </head>
   <body>
-    <div id="leave-comment-form">
-      Please share your feedback!
-      <form method="post" action="../comments">
-        <input name="email" type="email" placeholder="Your email adress">
-        <textarea name="content" wrap="soft" placeholder="Your comment"></textarea>
-        <input class="button" type="submit" placeholder="Submit!">
-      </form>
-    </div>
-    <div id="sidebar">
-      <div id="comment-list">
-        Comments:
-      </div>
+    <section id="leave-comment">
+      <span id="greeting">
+        Please log in to share your feedback!
+      </span>
+      <section class="g-signin2" data-onsuccess="onSignIn"></section>
+      <section id="show-on-sign-in">
+        <form id="leave-comment-form" method="post" action="../comments">
+          <textarea name="content" required wrap="soft" placeholder="Your comment"></textarea>
+          <input id="userTokenId" name="userTokenId" value="" type="hidden">
+          <input class="button" type="submit" placeholder="Submit!">
+        </form>
+        <a id="sign-out" class="button" href="leave-comment.html" onclick="signOut();">Sign out</a>
```

Comment:
`href="leave-comment.html"` causes the page to reload. Do you need the page to be reloaded for the signout to work? Right now it has a positive side effect of resetting the greeting message, but it causes an otherwise unnecessary page reload.

Reply:
Done